### PR TITLE
empty extension

### PR DESCRIPTION
**Pull request by kyessenov**
34K compiled
185K wee8-precompiled

Signed-off-by: Kuat Yessenov <kuat@google.com>

### CODE REVIEW

File: extensions/scaffold/plugin.h [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  explicit PluginRootContext(uint32_t id, std::string_view root_id)
+      : RootContext(id, root_id) {}
+
+  bool onConfigure(size_t) override;
+};
+
+// Per-stream context.
+class PluginContext : public Context {
+ public:
+  explicit PluginContext(uint32_t id, RootContext* root) : Context(id, root) {}
```
Comment (bianpengyuan) [inline extensions/scaffold/plugin.h:29]: Should we add available callbacks here as comment? Same comment to the root context.

Comment (kyessenov) [reply extensions/scaffold/plugin.h:29]: I think it's fine to leave as-is. I am mostly interested in performance measurements so want to keep it minimal.

**Review by bianpengyuan — APPROVED** [review]

File: extensions/scaffold/plugin.h (hunk 1 above)
Comment (mandarjog) [reply extensions/scaffold/plugin.h:29]: I think a different one (or same one with different #defines) that defines all callbacks is also important. You can add it as another plying though.

Comment (kyessenov) [reply extensions/scaffold/plugin.h:29]: Yeah, but they are easily discoverable since methods must be overridden and have default definitions.

**Pull request merged**